js/src/forum/components/SuspendUserModal.js

```diff
@@ -36,53 +43,7 @@ export default class SuspendUserModal extends Modal {
         <div className="Form">
           <div className="Form-group">
             <label>{app.translator.trans('flarum-suspend.forum.suspend_user.status_heading')}</label>
-            <div>
-              <label className="checkbox">
-                <input type="radio" name="status" checked={!this.status()} value="" onclick={withAttr('value', this.status)} />
-                {app.translator.trans('flarum-suspend.forum.suspend_user.not_suspended_label')}
-              </label>
-
-              <label className="checkbox">
-                <input
-                  type="radio"
-                  name="status"
-                  checked={this.status() === 'indefinitely'}
-                  value="indefinitely"
-                  onclick={withAttr('value', this.status)}
-                />
-                {app.translator.trans('flarum-suspend.forum.suspend_user.indefinitely_label')}
-              </label>
-
-              <label className="checkbox SuspendUserModal-days">
-                <input
-                  type="radio"
-                  name="status"
-                  checked={this.status() === 'limited'}
-                  value="limited"
-                  onclick={(e) => {
-                    this.status(e.target.value);
-                    m.redraw.sync();
-                    this.$('.SuspendUserModal-days-input input').select();
-                    e.redraw = false;
-                  }}
-                />
-                {app.translator.trans('flarum-suspend.forum.suspend_user.limited_time_label')}
-                {this.status() === 'limited' ? (
-                  <div className="SuspendUserModal-days-input">
-                    <input
-                      type="number"
-                      min="0"
-                      value={this.daysRemaining()}
-                      oninput={withAttr('value', this.daysRemaining)}
-                      className="FormControl"
-                    />
-                    {app.translator.trans('flarum-suspend.forum.suspend_user.limited_time_days_text')}
-                  </div>
-                ) : (
-                  ''
-                )}
-              </label>
-            </div>
+            <div>{this.formItems().toArray()}</div>
```

Comment:
I would reaaaaally like to see two list items instead, one for the radio options (a form group of its own) and the other for all form groups since they the two added fields should technically be their own form groups.